pulser/sequence.py

```diff
@@ -733,6 +733,19 @@ def __str__(self):
 
         return full
 
+    def get_duration(self):
+        """
+        Check current duration of Sequence.
+
+        Returns:
+            integer: The current duration in nanoseconds.
+        """
+        final_timeslots_per_channel = [0]
+        for ch in self._channels:
+            tf_of_last_element = self._last(channel=ch).tf
+            final_timeslots_per_channel.append(tf_of_last_element)
+        return max(final_timeslots_per_channel)
```

Comment:
This is better! Can we make it more compact, perhaps? See if you can do it in one line.